src/io/jesi/backpack/macros.cljc

```diff
@@ -108,14 +108,35 @@
               [form]))]
     (cons 'try (mapcat transform body))))
 
+;FIXME rename
+(defn shorthand* [keyfn symbols]
```

Comment:
Any idea what this should be called?

Reply:
I think the `*` suffix is fine. Otherwise `-shorthand`. Something like that.